_docs/tasks/security/secure-access-control.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Setup Secure Access Control
+overview: This task shows how to securely control access to a service using service accounts.
+
+order: 30
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows how to securely control access to a service,
+using the service accounts provided by Istio authentication.
+
+When mTLS authentication is enabled, the server authenticates the client and obtains the client's
```

Comment:
What about "When Istio auth is enabled", some customers may only know auth is enabled or not, but not know mTLS.

And use 'the server authenticates the client according to its certificate and obtain the client's service account...'

Reply:
Done.